IdentityServer/appsettings.json

```
@@ -8,22 +8,12 @@
     },
     "App": {
         "Self": {
-            "JwtAuthority": "http://localhost:5004",
+            "JwtAuthority": "",
             "DeleteTokenLifeTimeInDays": 90, //When a user request deletion of his account, this is the max days before the request is invalid; Default: 90
```

```suggestion
            "DeleteTokenLifeTimeInDays": 90,
```
No comments in JSON. thanks